### PR TITLE
Se agrego curl para  point via API

### PR DESCRIPTION
**Pull request by SofiaMoralesGonzalez1992**
## Description
Actualizamos los pasos para integrar  Point via API agregando el curl asi evitamos el contacto de support.

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [X] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/in-person-payments/mp-point/how-to-integrate.en.md [hunk 1]
```diff
@@ -83,21 +83,26 @@ It its very important that the code has the ability to handle the situation wher
...
-> * In order to be able to actually go ahead with this integration, you have to contact [support](https://www.mercadopago.com.ar/developers/en/support/), soy they can enable the Point integration settings in your Mercado Pago app. 
-
-Another way to integrate the Mercado Pago application with our Point devices is via API. 
-
-For this integration, first you need to configure the `device_name` from the Mercado Pago application. It helps us to identify the smartphone or tablet and link it to your Mercado Pago account. Like this, you´ll know to which device you have to send the payment_order. 
-
+> * This integration is only available for Android version 2.8.0 or higher.
+> * Not available for iOS.
+The other way to integrate with the Mercado Pago application to charge with our Point is through our APIs.
+for
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.en.md:94]: Eliminar

File: guides/in-person-payments/mp-point/how-to-integrate.en.md [hunk 2]
```diff
@@ -83,21 +83,26 @@ It its very important that the code has the ability to handle the situation wher
...
+for
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.en.md:95]: Eliminar

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 3]
```diff
@@ -87,18 +87,24 @@ En el artículo de [GitHub](https://github.com/mercadopago/point-android_integra
...
 > * Esta integración sólo esta disponible para Android versión 2.8.0 o superior.
 > * No esta disponible para iOS.
-> * Para poder utilizar esta integración es necesario que te comuniques con [soporte](https://www.mercadopago.com.ar/developers/es/support/) para que te habiliten las opciones de integraciones en la app de Mercado Pago.
-
 La otra forma de integrarse con la aplicación de Mercado Pago para cobrar con nuestro Point es mediante nuestras API.
-
-Para esta integración, primero es necesario configurar desde la aplicación de Mercado Pago el `device_name` . El mismo sirve para identificar tu celular o tablet y relacionarlo con tu cuenta de Mercado Pago. De esta manera, sabrás a que dispositivo enviar la orden de pago.
+<br>
+<br>
+Para poder integrarte, tienes que habilitar las opciones de itegraciones  en la app de Mercado Pago. Ejecuta el siguiente curl para realizarlo:
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:99]: - Hay un espacio extra luego de integraciones. 
- Falta la n en i**n**tegraciones

File: guides/in-person-payments/mp-point/how-to-integrate.es.md [hunk 4]
```diff
@@ -87,18 +87,24 @@ En el artículo de [GitHub](https://github.com/mercadopago/point-android_integra
...
+```curl
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.es.md:100]: El snippet no se visualiza correctamente. En español sí. 
Captura de Pantalla 2021-02-09 a la(s) 15 11 01

File: guides/in-person-payments/mp-point/how-to-integrate.pt.md [hunk 5]
```diff
@@ -85,20 +85,25 @@ Como referência é possível utilizar o código de exemplo e documentação que
...
-A outra forma de se integrar com a aplicação de Mercado Pago para cobrar com Point é mediante nossas APIs.
-
-Para esta integração, primeiro é necessário configurar na aplicação de Mercado Pago o `device_name` . O mesmo serve para identificar o celular ou tablet e relacioná-lo com a conta do Mercado Pago. Desta maneira, saberá a que dispositivo enviar a ordem de pagamento.
+> * Esta integração está disponível apenas para Android versão 2.8.0 ou superior.
+> * Não disponível para iOS.
+A outra forma de integração com o aplicativo Mercado Pago para cobrar com nosso Point é através de nossas APIs.
+para
+para
+Para fazer a integração, é necessário habilitar as opções de integração no aplicativo Mercado Pago. Execute o seguinte curl para fazer isso:
+`` `curl
```
Comment (abderossi) [inline guides/in-person-payments/mp-point/how-to-integrate.pt.md:99]: El snippet no se visualiza correctamente 
Captura de Pantalla 2021-02-09 a la(s) 15 10 52

**Review by abderossi — CHANGES_REQUESTED** [review]

**Review by abderossi — APPROVED** [review]

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**